html/input/demos/inputs.html

```diff
@@ -170,9 +170,9 @@
     <fieldset class="radio-set">
       <span class="radio-legend">Радио-кнопки:</span>
       <div class="radio-container">
-        <label for="ie" class="radio-label"><input class="radio" type="radio" name="browser" value="ie" id="ie"><span class="radio-title">Вариант 1</span></label>
-        <label for="opera" class="radio-label"><input class="radio" type="radio" name="browser" value="opera" id="opera"><span class="radio-title">Вариант 2</span></label>
-        <label for="firefox" class="radio-label"><input class="radio" type="radio" name="browser" value="firefox" id="firefox"><span class="radio-title">Вариант 3</span></label>
+        <label class="radio-label"><input class="radio" type="radio" name="browser" value="ie" id="ie"><span class="radio-title">Вариант 1</span></label>
+        <label class="radio-label"><input class="radio" type="radio" name="browser" value="opera" id="opera"><span class="radio-title">Вариант 2</span></label>
+        <label class="radio-label"><input class="radio" type="radio" name="browser" value="firefox" id="firefox"><span class="radio-title">Вариант 3</span></label>
```

Comment:
```suggestion
        <label class="radio-label">
          <input class="radio" type="radio" name="browser" value="ie" id="ie">
          <span class="radio-title">Вариант 1</span>
        </label>
        <label class="radio-label">
          <input class="radio" type="radio" name="browser" value="opera" id="opera">
          <span class="radio-title">Вариант 2</span>
        </label>
        <label class="radio-label">
          <input class="radio" type="radio" name="browser" value="firefox" id="firefox">
          <span class="radio-title">Вариант 3</span>
        </label>
```

Если это не мешает работе демки, я бы форматировал понятнее